journalbeat/magefile.go

```diff
@@ -22,20 +22,47 @@ package main
 import (
 	"context"
 	"fmt"
+	"strings"
 	"time"
 
 	"github.com/magefile/mage/mg"
 	"github.com/magefile/mage/sh"
+	"github.com/pkg/errors"
 
 	"github.com/elastic/beats/dev-tools/mage"
 )
 
 func init() {
 	mage.BeatDescription = "Journalbeat ships systemd journal entries to Elasticsearch or Logstash."
 
-	// TODO filter platforms
+	mage.Platforms = mage.Platforms.Filter("linux !linux/ppc64 !linux/mips64")
 }
 
+var (
+	deps = map[string]func() error{
+		"linux/386":      installLinux386,
+		"linux/amd64":    installLinuxAMD64,
+		"linux/arm64":    installLinuxARM64,
+		"linux/armv5":    installLinuxARMLe,
```

Comment:
`s/installLinuxs390x/installLinuxS390X/`

Actually, can you just make everything after "Linux" all-caps for each function. Most everything there is an acronym so with Go's naming conventions I think they should all be in caps. Plus there's a mix of "Le" and "le" which is inconsistent.